url.bs

```diff
@@ -681,15 +681,14 @@ steps:
 
  <li><p>Let <var>last</var> be the last <a for=list>item</a> in <var>parts</var>.
 
- <li><p>If parsing <var>last</var> as an <a lt="IPv4 number parser">IPv4 number</a> does not
- return failure, then return true.
-
  <li>
   <p>If <var>last</var> is non-empty and contains only <a>ASCII digits</a>, then return true.
```

Comment:
Since this no longer has a note, the `<p>` needs to be on the same line as `<li>`